flink/src/main/scala/ru/itclover/tsp/mappers/ProcessorCombinator.scala

```diff
@@ -0,0 +1,67 @@
+package ru.itclover.tsp.mappers
+import org.apache.flink.streaming.api.scala.function.ProcessWindowFunction
+import org.apache.flink.streaming.api.windowing.windows.Window
+import org.apache.flink.util.Collector
+import ru.itclover.tsp.mappers.PatternProcessor
+import ru.itclover.tsp.v2.PState
+
+import scala.reflect.ClassTag
+//
+//import org.apache.flink.api.common.functions.{AbstractRichFunction, RichFlatMapFunction, RuntimeContext}
```

Comment:
Dead comments, no need to store it (it's in history)

Reply:
Fixed